lock-api/src/main/java/com/palantir/lock/client/DialogueAdaptingConjureTimelockService.java

```diff
@@ -36,15 +39,23 @@
 
 public class DialogueAdaptingConjureTimelockService implements ConjureTimelockService {
     private final ConjureTimelockServiceBlocking dialogueDelegate;
+    private final ConjureTimelockServiceBlockingMetrics conjureTimelockServiceBlockingMetrics;
 
-    public DialogueAdaptingConjureTimelockService(ConjureTimelockServiceBlocking dialogueDelegate) {
+    public DialogueAdaptingConjureTimelockService(ConjureTimelockServiceBlocking dialogueDelegate,
+            ConjureTimelockServiceBlockingMetrics conjureTimelockServiceBlockingMetrics) {
         this.dialogueDelegate = dialogueDelegate;
+        this.conjureTimelockServiceBlockingMetrics = conjureTimelockServiceBlockingMetrics;
     }
 
     @Override
     public ConjureStartTransactionsResponse startTransactions(AuthHeader authHeader, String namespace,
             ConjureStartTransactionsRequest request) {
-        return dialogueDelegate.startTransactions(authHeader, namespace, request);
+        long timeCreated = System.currentTimeMillis();
```

Comment:
What happens if a request takes 3.7 ms?

I'd suggest we use `Instant.now()` instead 😄